variables/_colors.pcss

```diff
@@ -87,23 +86,17 @@
 	/* -----------------------------------------------------------------------------
 	 * Colors - Element UI
 	 * ----------------------------------------------------------------------------- */
-	--tec-color-background-primary-multiday: rgba(var(--tec-color-accent-primary), 0.24);
-	--tec-color-background-primary-multiday-hover: rgba(var(--tec-color-accent-primary), 0.34);
-	--tec-color-background-secondary-multiday: rgba(var(--tec-color-accent-secondary), 0.24);
-	--tec-color-background-secondary-multiday-hover: rgba(var(--tec-color-accent-secondary), 0.34);
-	--tec-color-accent-primary-week-event: rgba(var(--tec-color-accent-primary), 0.1);
-	--tec-color-accent-primary-week-event-hover: rgba(var(--tec-color-accent-primary), 0.2);
-	--tec-color-accent-primary-week-event-featured: rgba(var(--tec-color-accent-primary), 0.04);
-	--tec-color-accent-primary-week-event-featured-hover: rgba(var(--tec-color-accent-primary), 0.14);
+	--tec-color-background-primary-multiday: rgba(51, 74, 255, 0.24);
+	--tec-color-background-primary-multiday-hover: rgba(51, 74, 255, 0.34);
+	--tec-color-background-secondary-multiday: rgba(20, 24, 39, 0.24);
+	--tec-color-background-secondary-multiday-hover: rgba(20, 24, 39, 0.34);
+	--tec-color-accent-primary-week-event: rgba(51, 74, 255, 0.1);
+	--tec-color-accent-primary-week-event-hover: rgba(51, 74, 255, 0.2);
+	--tec-color-accent-primary-week-event-featured: rgba(51, 74, 255, 0.04);
+	--tec-color-accent-primary-week-event-featured-hover: rgba(51, 74, 255, 0.14);
 	--tec-color-background-secondary-datepicker: var(--tec-color-background-secondary);
 	--tec-color-accent-primary-background-datepicker: var(--tec-color-accent-primary-background);
 
-	/* -----------------------------------------------------------------------------
-	 * Colors - Theme-specific
-	 * ----------------------------------------------------------------------------- */
-
-	--tec-color-twentyseventeen-dark-background: #222;
```

Comment:
any reason why you're removing this here? the var `--color-twentyseventeen-dark-background` at the bottom of this file uses this color var.

Reply:
Ah, that should have moved too - it's only used in one file in ECP.